src/status_im/communities/core.cljs

```diff
@@ -558,16 +558,22 @@
 (rf/defn request-to-join-accepted
   {:events [::request-to-join-accepted]}
   [{:keys [db] :as cofx} community-id request-id response-js]
-  (rf/merge cofx
-            {:db (update-in db [:communities/requests-to-join community-id] dissoc request-id)}
-            (handle-response response-js)))
+  (rf/merge
+   cofx
+   {:db             (update-in db [:communities/requests-to-join community-id] dissoc request-id)
+    :dispatch-later [{:dispatch [:activity-center.notifications/mark-as-read request-id]
+                      :ms       100}]}
+   (handle-response response-js)))
 
 (rf/defn request-to-join-declined
   {:events [::request-to-join-declined]}
   [{:keys [db] :as cofx} community-id request-id response-js]
-  (rf/merge cofx
-            {:db (update-in db [:communities/requests-to-join community-id] dissoc request-id)}
-            (handle-response response-js)))
+  (rf/merge
+   cofx
+   {:db             (update-in db [:communities/requests-to-join community-id] dissoc request-id)
+    :dispatch-later [{:dispatch [:activity-center.notifications/mark-as-read request-id]
```

Comment:
could mark-as-read be used here directly without dispatches? or at least `dispatch-later` looks redundant here

Reply:
still open question

Reply:
`mark-as-read` could not be used directly. 

Cleaned up the redundant code in `dispatch-later` 👍.  These events (`accept-request-to-join-pressed` & `decline-request-to-join-pressed`) are used in communities (apart from Activity Center).